library/junos_install_os

```diff
@@ -155,7 +161,20 @@ import logging
 import re
 import os
 from distutils.version import LooseVersion
+from ansible.module_utils.six import string_types
+from ansible.module_utils.basic import *
 
+def _convert_to_boolean(val):
+    if val is None or isinstance(val, bool):
+        return val
```

Comment:
I think we can keep this function functionality inside the for loop itself for 2 reasons:
1> we are returning None in first if condition and None is not a boolean
2> Raising a typeError just to capture other data can be removed if we have code in for loop itself